### PR TITLE
Phoenix Hard-fork to occur at the same block as Aztlan - to "fix it"

### PR DESCRIPTION
**Pull request by bobsummerwill**

### CODE REVIEW

File: _specs/ecip-aztlan-fix.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+lang: en
+ecip: Unassigned
```
Comment (sorpaas) [inline _specs/ecip-aztlan-fix.md]: Assign number ECIP-1078.

**Comment by bobsummerwill** [comment]
ECIP Editors: @soc1c @YazzyYaz @meowsbits @sorpaas @BelfordZ:
Please can you review/comment/merge?  I think this is ready to go.  Thanks!

File: _specs/ecip-1078.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- Multi-Geth
+- Hyperledger Besu
+
+The author is unaware if any of these clients have implemented [EIP-1706](https://eips.ethereum.org/EIPS/eip-1706), but it is a very simple change, analogous to [EIP-2200](https://eips.ethereum.org/EIPS/eip-2200), which all the
+codebases have already implemented.
+
+Similarly [ECIP-1080](https://github.com/ethereumclassic/ECIPs/issues/266) has not been implemented in any of these codebases, but is just a "remix" of code which they have already implemented.
+
+
+## Copyright
```
Comment (sorpaas) [inline _specs/ecip-1078.md]: To comply with the Apache-2.0 license, please include the content of the NOTICE file here: https://corepaper.org/NOTICE

**Comment by bobsummerwill** [comment]
@sorpaas Is this blocked from merging purely on your review?  Please can you unblock and merge please?  Thank you.

**Comment by bobsummerwill** [comment]
@sorpaas Copyright language corrected.  This good now?

**Pull request merged**